### PR TITLE
Test/p2p message endpoint

### PR DESCRIPTION
**Pull request by mgcouchbase**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- http://uberjenkins.sc.couchbase.com:8080/view/testkit-testing/job/ios-p2p/11/
-http://uberjenkins.sc.couchbase.com:8080/view/testkit-testing/job/net-ios-android/5/
http://uberjenkins.sc.couchbase.com:8080/view/testkit-testing/job/android-p2p/5/testReport/

### CODE REVIEW

File: CBLClient/Query.py [hunk 1]
```diff
@@ -447,14 +447,12 @@ def removeChangeListener(self, query, change_listener):
         args.setMemoryPointer("changeListener", change_listener)
         return self._client.invokeMethod("query_removeChangeListener", args)
 
-    def query_selectAll(self, database):
+    def query_select_all(self, database):
```
Comment (euniceatcouchbase) [inline CBLClient/Query.py:450]: please sync up your branch with the latest master, this change will break some tests

File: testsuites/CBLTester/p2p_tests/conftest.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import datetime
 import pytest
 
-from keywords.utils import log_info, clear_resources_pngs, set_device_enabled
+from keywords.utils import log_info, clear_resources_pngs
```
Comment (euniceatcouchbase) [inline testsuites/CBLTester/p2p_tests/conftest.py:5]: again, this will wipe out the device/emulator combination changes

**Review by euniceatcouchbase — CHANGES_REQUESTED** [review]
Please Please sync up your branch with master before sending out a PR.

**Review by euniceatcouchbase — CHANGES_REQUESTED** [review]

File: testsuites/CBLTester/p2p_tests/conftest.py [hunk 3]
```diff
@@ -128,9 +127,8 @@ def params_from_base_suite_setup(request):
             log_info("Downloading TestServer ...")
             # Download TestServer app
             testserver.download()
-
             # Install TestServer app
-            if device_enabled:
+            if run_on_device:
```
Comment (euniceatcouchbase) [inline testsuites/CBLTester/p2p_tests/conftest.py:131]: this still rollback the device support. Could you merge the master to your branch and solve the conflicts?

**Review by euniceatcouchbase — CHANGES_REQUESTED** [review]

File: testsuites/CBLTester/p2p_tests/conftest.py (hunk 3 above)
Comment (mgcouchbase) [reply testsuites/CBLTester/p2p_tests/conftest.py:131]: I updated and resolved the conflicts, I changed this condition intentionally

Comment (euniceatcouchbase) [reply testsuites/CBLTester/p2p_tests/conftest.py:131]: I don't understand why you make this change. if you intentionally to rollback to the origin, only want to run on-device or not-on-device for all devices, you will need to clean out the code
"device_enabled_list": device_enabled_list,
and update all p2p jobs back to original states.

**Review by euniceatcouchbase — APPROVED** [review]

**Review by sridevi-15 — APPROVED** [review]

**Pull request merged**